modules/openapi-generator/src/main/resources/swift5/libraries/urlsession/URLSessionImplementations.mustache

```diff
@@ -618,6 +620,24 @@ private class FormURLEncoding: ParameterEncoding {
     }
 }
 
+private class OctetStreamEncoding: ParameterEncoding {
```

Comment:
thanks for the PR. can you please add a line or 2 explaining what this function does?